compiler/resolution/wrappers.cpp

```diff
@@ -577,12 +576,6 @@ static void addArgCoercion(FnSymbol*  fn,
       // Don't insert a readFE or readFF when deleting a sync/single.
```

Comment:
In this branch that looks for a function named "free": It looks like its trying to avoid deleting the sync. I don't recall ever seeing a "free" function lying around, could it be that this branch is unused?

We don't need to address it in this PR, but something I was wondering about.